### PR TITLE
i added function for erf tensorflow frontend and also added test function for it.

### LINKED ISSUE #19786: erf

### PR DESCRIPTION
**Pull request by sharjeelnawaz8182**
# PR Description

## Related Issue

Close #19786

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: .idea/ivy.iml
Comment (YushaArif99) [inline .idea/ivy.iml:1]: can you kindly revert these changes 🙂

File: .idea/misc.xml
Comment (YushaArif99) [inline .idea/misc.xml:1]: these as well 🙂

File: ivy/functional/backends/numpy/elementwise.py
Comment (YushaArif99) [inline ivy/functional/backends/numpy/elementwise.py:1]: these as well 🙂

File: ivy/functional/frontends/mxnet/numpy/mathematical_functions.py
Comment (YushaArif99) [inline ivy/functional/frontends/mxnet/numpy/mathematical_functions.py:1]: these as well 🙂

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py [hunk 1]
```diff
@@ -11,7 +11,9 @@
     _get_second_matrix_and_dtype,
     _get_dtype_value1_value2_axis_for_tensordot,
 )
-from ivy_tests.test_ivy.test_functional.test_experimental.test_core.test_elementwise import ldexp_args
+from ivy_tests.test_ivy.test_functional.test_experimental.test_core.test_elementwise import (
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:14]: this as well 🙂

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:1]: these as well 🙂

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 2]
```diff
@@ -1823,10 +1823,10 @@ def test_tensorflow_floormod(
         fn_tree=fn_tree,
         on_device=on_device,
         x=x[0],
-        y=x[1],
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:1826]: this as well 🙂

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @sharjeelnawaz8182 

Apologies for the delay in review. Have been a little swamped with work recently and so havent had the time to review any PRs 😅 

With regards to the PR itself, it seems like you've added quite a lot of unwanted changes. Could you kindly revert these so that the only changes are of the `erf` function? 
Also, could you kindly make sure the `PR compliance checks` are passing for this PR? 

I am unable to view the CI tests at this moment but they should run once the merge conflicts are resolved. Feel free to re-request a review once all relevant tests are passing. Thanks! 🙂

**Comment by sharjeelnawaz8182** [comment]
okay

**Pull request closed without merging**